ydb/docs/en/core/reference/ydb-cli/scripting-yql.md

```diff
@@ -1,3 +1,11 @@
+
+{% note warning %}
+
+This command is deprecated.
+The preferred way to run queries in {{ ydb-short-name }} CLI is to use the [`{{ ydb-cli }} sql`](sql.md) command.
+
+{% endnote %}
```

Comment:
IIRC, there has already been the same message above. If so, make an include file out of it instead of copy-pasting.